src/app/games/lodestone/lodestone.component.ts

```diff
@@ -475,14 +485,26 @@ export class LodestoneComponent
         return lodestoneInfo;
     }
 
-    private wasLodestoneDroppedThenCrumbled(coord: Coord): boolean {
+    private wasLastMoveLodestone(coord: Coord): boolean {
         if (this.node.parent.isPresent()) {
             return this.node.previousMove.get().coord.equals(coord);
         } else {
             return false;
         }
     }
 
+    private isCrumbledLodestone(coord: Coord): boolean {
+        if (this.node.parent.isPresent()) {
+            const state: LodestoneState = this.node.parent.get().gameState;
+            const piece: LodestonePiece = state.getPieceAt(coord);
+            return piece.isLodestone() &&
+                   piece.owner === state.getCurrentOpponent() &&
+                   state.turn === this.getState().turn;
```

Comment:
cette ligne rend cette fonction tout le temps faux. Tu peux remplacer isCrumbledLodestone par `return false` et ça fera la même chose.
Ça ne résous pas le bug : après que le mouvement soit fait, la lodestone est transformée en pièce.
Enlever le `state.turn == this.getState().turn` "résous" le bug partiellement : après le move, la lodestone est bonne. Mais PENDANT le move, il y a la revenante.